kartothek/io/eager.py

```diff
@@ -628,6 +699,18 @@ def create_empty_dataset_header(
 
 @default_docs
 @normalize_args
+@deprecate_parameters_if_set(
+    get_parameter_type_change_deprecation_warning(
+        from_type="Optional[ParquetSerializer]",
+        to_type="Optional[DataFrameSerializer]",
+        deprecated_in="5.3",
+        changed_in="6.0",
+    ),
+    "df_serializer",
```

Comment:
Are we really removing this? I thought this was added as feature with #404

Reply:
get_parameter_type_change_deprecation_warning
type change ParquetSerializer -> DataFrameSerializer
<img width="617" alt="image" src="https://user-images.githubusercontent.com/84850320/143277329-70f2685e-cb79-4c03-b4fb-91554530a8a0.png">
https://github.com/JDASoftwareGroup/kartothek/pull/472/files#diff-28fbe6ec73d8097d11b7c93cb8decfeb6dbed57a5729ac3a90501dbac2abf67a

Reply:
My bad, the linked PR add the Serializer to the cube interface. Thought it would expose it in general.